### DISCUSSION TITLE
Data types

### DISCUSSION THREAD

**Discussion by ssvendsen** [apache/iotdb#4297] [Q&A] [answered]
Hi. Are there any plans of supporting 8- and 16-bit integers (byte and short), and unsigned integers natively?
https://iotdb.apache.org/UserGuide/Master/Data-Concept/Data-Type.html#data-type
Thank you.

**Comment by HTHou** [comment, marked as answer]
Hi, extending more data types is always the reasonable feature I think! 
Welcome to submit an issue to our community. https://issues.apache.org/jira/projects/IOTDB/issues
Thank you.

**Reply by ssvendsen** [reply]
Thank you. I have created https://issues.apache.org/jira/browse/IOTDB-1921
